site/_posts/2017-03-12-marching-ahead-embedded-ansible.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Last Week in ManageIQ: March-ing ahead with the embedded Ansible integration
+author: lgalis
+date: 2017-03-12 21:33:30 -04:00
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+[Laura](https://github.com/lgalis) here bringing you this week's edition of Last Week in ManageIQ!
+
+## Featured
+A busy week again with a lot of progress towards our embedded Ansible Tower features, along with refactoring of existing code, as we move closer to the 
+development complete milestone for our Fine release. Last week, there were 293 commits in 184 PRs across our combined repositories: 
+ - 73 PRs in the [manageiq](manageiq PRs merged) repo
+ - 84 in the [manageiq-ui-classic](manageiq-ui-classic PRs merged) repo
+ - 17 in the [manageiq-ui-service](manageiq-ui-service PRs merged) repo 
```

Comment:
note usage of round and square brackets,

either: `[link text](url)`

or: `[link text][link id]`
`[link id]: url`